src/modals/index.js

```diff
@@ -45,15 +49,26 @@ const StyledContainer = styled.div`
   justify-content: center;
 `;
 
+const reduxProps = ({ modal }) => ({
+  modal: modal.modal,
+});
+
 class Modal extends Component {
+  propTypes = {
```

Comment:
@samelliott89 I noticed I was seeing a warning: propTypes was defined as an instance property on Modal. Use a static property to define propTypes instead.

Reply:
I think we can add `static propTypes = {}` and that should fix, just tested locally.

Reply:
Hey @samelliott89  - yes you're right, it just needs to be declared as static. I'll add that. Thanks a lot! Cheers, will add these